src/Console/Worker.php

```diff
@@ -57,11 +57,11 @@ protected function getHelp(): string
     -s|--spawn              Spawn an additional worker
 
 Examples
-	bin/console daemon start -f
-		Starts the daemon in the foreground
+	bin/console worker -n
+		Starts the worker without executing the cronob
```

Comment:
Besides the typo, what does this mean to not execute the cronjob?

Reply:
It doesn't execute https://github.com/friendica/friendica/blob/develop/src/Worker/Cron.php#L22 , whatever impact that has :-) 

maybe @annando can explain `no_cron` and `spawn` in a more precise way and I can take it for the help-texte

Reply:
Normally the system will automatically execute a call to the `cron` tasks on startup. With this parameter it doesn't do so.

Reply:
And why is it possible? What benefit do we get?

And is `spawn` correctly a parameter to spawn **another** worker process beside the current one?

Reply:
Then it should read something like this:
```suggestion
		Starts the worker without executing other recurring tasks
```